cdap-data-fabric-tests/src/test/java/co/cask/cdap/data/stream/InMemoryStreamCoordinatorTest.java

```diff
@@ -34,21 +44,42 @@
 
   private static Injector injector;
 
+  private static TransactionManager transactionManager;
+  private static DatasetService datasetService;
+
   @BeforeClass
   public static void init() {
     injector = Guice.createInjector(
       new ConfigModule(),
       new DiscoveryRuntimeModule().getInMemoryModules(),
       new DataFabricModules().getInMemoryModules(),
+      new DataSetServiceModules().getInMemoryModule(),
```

Comment:
Yes I'll do that, although it requires that a lot of tests do the binding to the dummy ``StreamMetastore`` implementation. And there is no existing module where we can put that.